extensions/arc/src/localizedConstants.ts

```diff
@@ -246,6 +248,8 @@ export function copyValueToClipboard(valueName: string): string { return localiz
 export const pgConnectionRequired = localize('arc.pgConnectionRequired', "A connection is required to show and set database engine settings.");
 export const miaaConnectionRequired = localize('arc.miaaConnectionRequired', "A connection is required to list the databases on this instance.");
 export const couldNotFindControllerRegistration = localize('arc.couldNotFindControllerRegistration', "Could not find controller registration.");
+export const dropMultipleExtensions = localize('arc.dropMultipleExtensions', "Only drop one extension at a time.");
```

Comment:
Maybe a more thorough description here - something like `Currently dropping another extension, try again once that is completed`. 

Some alternatives to consider : 
- Queuing up concurrent requests and executing them sequentially instead. 
- Having checkboxes for each row and then a top level delete extensions button (that way you could disable the button while a delete is in process too)

Reply:
Updated string. I like the idea of the checkboxes, will bring to attention of designer.